### ISSUE TITLE
[BUG] css module compose from within same file results in false positive

### ISSUE THREAD

**Issue by lulzhipsters** [malinindev/check-unused-css#83]
## Summary
When a class exists and is referenced, but is also referenced in another class using css-module's `compose: ` syntax, check-unused-css reports that the referenced class actually doesn't exist in the css module.

### CSS Module (Test.module.css)
``` css
.definedClass {
  color: red;
}

.composingClass {
  composes: definedClass;
  font-size: 14px;
}

```

### Component (Test.tsx)
```tsx
import styles from "./Test.module.css";

export function Test() {
  return <div className={styles.definedClass}>Hello</div>;
}

```

### Command
`npx check-unused-css@latest`

### Result
```bash
Checking for unused CSS classes...

Found 1 classes used in source files but non-existent in CSS:

  Test.tsx:4:33 - .definedClass

Found 1 classes defined in CSS but unused in source files:

  Test.module.css:5:1 - .composingClass

Tip: run check-unused-css --remove to delete the safely-removable ones automatically (preview + confirmation before any file is touched).
```

### Expected Result
```
Checking for unused CSS classes...

Found 1 classes defined in CSS but unused in source files:

  Test.module.css:5:1 - .composingClass

Tip: run check-unused-css --remove to delete the safely-removable ones automatically (preview + confirmation before any file is touched).
```

**Comment by malinindev** [comment]
@lulzhipsters, thank you for the report! Fixed in v5.1
